src/platform/user/profile/vet360/components/AddressField/address-schemas.js

```diff
@@ -5,6 +5,9 @@ import cloneDeep from 'platform/utilities/data/cloneDeep';
 
 import { ADDRESS_FORM_VALUES, USA } from 'vet360/constants';
 
+const blockURLsRegEx =
+  '^((?!https?:?/?/?|www\\.|\\.co|\\.net|\\.gov|\\.edu|\\.org).)*$';
+
 // make an object of just the military state codes and names
 const MILITARY_STATES = Object.entries(ADDRESS_DATA.states).reduce(
```

Comment:
I know it's unrelated to the PR, but if you want, you can import this directly from `vets-json-schema` - https://github.com/department-of-veterans-affairs/vets-json-schema/blob/master/src/common/address.js#L252-L256